src/rocketmq_client.erl

```diff
@@ -129,7 +129,7 @@ handle_response(<<>>, State) ->
 handle_response(Bin, State = #state{requests = Reqs, last_bin = LastBin}) ->
     case rocketmq_protocol_frame:parse(<<LastBin/binary, Bin/binary>>) of
         {undefined, undefined, Bin1} ->
-            {nodelay, State#state{last_bin = Bin1}, hibernate};
+            {noreply, State#state{last_bin = Bin1}, hibernate};
```

Comment:
We need to print the content of `last_bin` when the call fails with `timeout` reason, as it may contains some incorrect packets.